docs/core-concepts/index.md

```diff
@@ -0,0 +1,36 @@
+# Core Concepts
+
+These are some of the core concepts that you should be familiar with before going over the [Getting Started Guide](get-started/index.md).
+
+## GitOps
+
+GitOps is a way of managing your infrastructure and applications so that whole system is described declaratively and version controlled(most likely in a Git repository), and having an automated process that ensure that the deployed environment matches the state specified in a repository.
+
+For more information, take a look at [this documentation](https://www.gitops.tech/#what-is-gitops).
+
+## Sources
+
+Sources in Flux represent sources of configuration like GitRepositories and Helm Releases.
+
+For more information, take a look at [this documentation](components/source/gitrepositories.md)
```

Comment:
```suggestion
A *Source* defines the origin of a source and the requirements to obtain
it (e.g. credentials, version selectors). For example, the latest `1.x` tag
available from a Git repository over SSH.

Sources produce an artifact that is consumed by other Flux elements to perform
actions, like applying the contents of the artifact on the cluster. A source
may be shared by multiple consumers to deduplicate configuration and/or storage.

The origin of the source is checked for changes on a defined interval, if
there is a newer version available that matches the criteria, a new artifact
is produced.

All sources are specified as Custom Resources in a Kubernetes cluster, examples
of sources are `GitRepository`, `HelmRepository` and `Bucket` resources. 
```

Reply:
Thanks! Do you think we need to explain what an artifact is too?